### PR TITLE
Doc for masking for windows source template

### PR DESCRIPTION
**Pull request by sumoanema**
## Purpose of this pull request

This pull request is to add a doc for windows source template masking, which will be different from conventional masking process which we have for other source templates.

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions, updating sections
- [X] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

## Ticket (if applicable)
https://sumologic.atlassian.net/browse/SUMO-241150

### CODE REVIEW

File: docs/send-data/opentelemetry-collector/remote-management/processing-rules/mask-rules.md [hunk 1]
```diff
@@ -10,6 +10,10 @@ description: Create a mask rule to replace an expression with a mask string.
 
 <p><a href="/docs/beta"><span className="beta">Beta</span></a></p>
 
+:::note
+For masking logs for Windows Source Template please refer [this](mask-rules-windows.md) doc.
```
Comment (JV0812) [inline docs/send-data/opentelemetry-collector/remote-management/processing-rules/mask-rules.md:14]:
```suggestion
This document do not support masking logs for Windows source template. Refer to [Mask Rules for Windows Source Template](mask-rules-windows.md) to mask logs for Windows source template.
```

File: docs/send-data/opentelemetry-collector/remote-management/source-templates/windows/index.md [hunk 2]
```diff
@@ -65,7 +65,7 @@ import ProcMetrics from '../../../../../reuse/apps/opentelemetry/process-metric-
 <ProcMetrics/>
 
 - **Scan Interval**. The frequency at which the source is scanned.
-- **Processing Rules**.  You can add processing rules for logs/metrics collected. To learn more, refer to [Processing Rules](../../processing-rules/index.md).
+- **Processing Rules**.  You can add processing rules for logs/metrics collected. To learn more, refer to [Processing Rules](../../processing-rules/index.md). For masking windows event logs please refer to [this](../../processing-rules/mask-rules-windows.md) doc
```
Comment (JV0812) [inline docs/send-data/opentelemetry-collector/remote-management/source-templates/windows/index.md:68]:
```suggestion
- **Processing Rules**.  You can add processing rules for logs/metrics collected. To learn more, refer to [Processing Rules](../../processing-rules/index.md). For masking windows event logs,  refer to [Mask Rules for Windows Source Template](mask-rules-windows.md).
```

**Review by JV0812 — CHANGES_REQUESTED** [review]
Minor fixes.

**Review by JV0812 — APPROVED** [review]
LGTM

**Review by jpipkin1 — APPROVED** [review]
Approved, @sumoanema. I made some minor changes (see commit "[Updates from review](https://github.com/SumoLogic/sumologic-documentation/pull/4686/commits/40e4d21a4a1aa73fa31d949edc3b9352f7ec9b7c)").

**Pull request merged**